packages/lodestar/src/chain/blocks/verifyBlock.ts

```diff
@@ -317,5 +316,24 @@ export async function verifyBlockStateTransition(
     });
   }
 
+  // All checks have passed, if this is a merge transition block we can log
+  if (isMergeTransitionBlock) {
+    logOnPowBlock(chain, block);
+  }
+
   return {postState, executionStatus};
 }
+
+function logOnPowBlock(chain: VerifyBlockModules, block: allForks.SignedBeaconBlock): void {
```

Comment:
I think it would be safer to have
```ts
function logOnPowBlock(chain: VerifyBlockModules, block: bellatrix.SignedBeaconBlock): void
```
and do the casting right below `isMergeTransitionBlock` where it's clear that the casting is safe